### PR TITLE
Use Markdown (MDX) 

### PR DESCRIPTION
**Pull request by shirgoldbird**
Use MDX for blog posts as well as pages using our standard template.

### CODE REVIEW

**Comment by shirgoldbird** [comment]
@daredoes @stevestreza do either of you have some time to look at this in the next few days? 🙏🏻  Hopefully shouldn't need any more work but I'd love a second set of eyes.

File: gatsby-node.js [hunk 1]
```diff
@@ -4,13 +4,15 @@ const { createFilePath } = require(`gatsby-source-filesystem`)
 exports.onCreateNode = ({ node, getNode, actions }) => {
   const { createNodeField } = actions
 
-  if (node.internal.type === `MarkdownRemark`) {
-    const slug = createFilePath({ node, getNode, basePath: `pages` })
-
+  if (node.internal.type === `Mdx`) {
+    let slug = createFilePath({ node, getNode, basePath: `pages` })
+    if (node.frontmatter.template === 'blogpost') {
```
Comment (stevestreza) [inline gatsby-node.js]: nit: use backticks for consistency

File: src/markdown-pages/blog.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+slug: /contact
+title: Contact
```
Comment (stevestreza) [inline src/markdown-pages/blog.mdx]: I think slug and title need to be changed

File: gatsby-node.js (hunk 1 above)
Comment (shirgoldbird) [reply gatsby-node.js]: fixed!

File: src/markdown-pages/blog.mdx (hunk 2 above)
Comment (shirgoldbird) [reply src/markdown-pages/blog.mdx]: OOPS, good catch

**Pull request merged**